architecture/topics/router_environment_variables.adoc

```diff
@@ -81,18 +69,46 @@ endif::[]
 |`ROUTER_CIPHERS` | intermediate  | Specify the set of xref:ciphers[ciphers] supported by bind.
 |===
 
-[NOTE]
-====
-Some of the timeouts that we expose interact casuing the total timeout to be longer then specified.
-
-For example:
-ROUTER_SLOWLORIS_HTTP_KEEPALIVE adjusts "timeout http-keep-alive", but haproxy still waits on the "tcp-request inspect-delay" which we set to 5s. So the overall timeout will be ROUTER_SLOWLORIS_HTTP_KEEPALIVE + 5s.
-====
-
 [NOTE]
 ====
 If you want to run multiple routers on the same machine, you must change the
 ports that the router is listening on, `ROUTER_SERVICE_SNI_PORT` and
 `ROUTER_SERVICE_NO_SNI_PORT`. These ports can be anything you want as long as
 they are unique on the machine. These ports will not be exposed externally.
 ====
+
+[[time-units]]
+*Router timeout variables*
+
+`TimeUnits` are represented by a number followed by the unit: `us`
+*(microseconds), `ms` (milliseconds, default), `s` (seconds), `m` (minutes), `h`
+*(hours), `d` (days).
+
+The regular expression is: [1-9][0-9]*(us\|ms\|s\|m\|h\|d)
+
+[cols="2,2,6"]
+|===
+|`ROUTER_BACKEND_CHECK_INTERVAL` | 5000ms | Length of time between subsequent "liveness" checks on backends.
+|`ROUTER_CLIENT_FIN_TIMEOUT` | 1s | Controls the TCP FIN timeout period for the client connecting to the route. If the FIN sent to close the connection is not answered within the given time, HAProxy will close the connection anyway.  This is harmless if set to a low value and uses fewer resources on the router.
+|`ROUTER_DEFAULT_CLIENT_TIMEOUT`| 30s | Length of time within which a client has to acknowledge or send data.
+|`ROUTER_DEFAULT_CONNECT_TIMEOUT`| 5s | The maximum connect time.
+|`ROUTER_DEFAULT_SERVER_FIN_TIMEOUT` | 1s | Controls the TCP FIN timeout from the router to the pod backing the route.
+|`ROUTER_DEFAULT_SERVER_TIMEOUT`| 30s | Length of time within which a server has to acknowledge or send data.
+|`ROUTER_DEFAULT_TUNNEL_TIMEOUT` | 1h | Length of time till which TCP or WebSocket connections will remain open. If you have websockets/tcp
+connections (and any time HAProxy is reloaded), the old HAProxy processes
+will "linger" around for that period.
+|`ROUTER_SLOWLORIS_HTTP_KEEPALIVE`| 300s | Set the maximum time to wait for a new HTTP request to appear. If this is set too low, it can confuse browsers and applications not expecting a small `keepalive` value.
```

Comment:
Would be nice if here we add something like `see note below` or something to that effect.